Sources/Web3Wallet/Web3WalletImports.swift

```diff
@@ -0,0 +1,3 @@
+#if !CocoaPods
+@_exported import WalletConnectPairing
```

Comment:
also need to remove all packages imports from all files

Reply:
Cocoapods putting all sources in single framework, so it doesn't know what is Auth, WalletConnectSign etc

Need to remove all 

```swift
import WalletConnectSign
import Auth
```
And imports should be
```swift
@_exported import Auth
@_exported import WalletConnectSign
```

Agree it looks like a little wired but supporting both SPM and Cocoapods is a kind of pain